tools/compare.py

```diff
@@ -48,6 +48,14 @@ def create_parser():
              "of repetitions. Do note that only the display is affected. "
              "Internally, all the actual runs are still used, e.g. for U test.")
 
+    parser.add_argument(
+        '--no-color',
+        dest='color',
+        default=True,
```

Comment:
The default behavior was to always use color, since the `use_color` argument was not used when calling `generate_difference_report`:
https://github.com/google/benchmark/pull/990/files#diff-56ab0b26bdd1e3b37cf2b787c63783d7R248-R250
, which defaults to `true`:
https://github.com/google/benchmark/blob/master/tools/gbench/report.py#L210-L216